test/AutobahnTestApp/project.json

```diff
@@ -0,0 +1,56 @@
+﻿{
+    "dependencies": {
+        "Microsoft.AspNetCore.WebSockets.Server": { "target": "project" },
+        "Microsoft.NETCore.App": {
+            "version": "1.0.0",
+            "type": "platform"
+        },
+
+        "Microsoft.AspNetCore.Diagnostics": "1.1.0-*",
+        "Microsoft.AspNetCore.Server.IISIntegration": "1.1.0-*",
+        "Microsoft.AspNetCore.Http.Abstractions": "1.1.0-*",
```

Comment:
yeah... it was added by VS while I was trying to `Ctrl-.` for `Response.WriteAsync`...